Sources/Target/FreeBSD/Process.cpp

```diff
@@ -40,7 +40,7 @@ Process::Process()
     : super(), _softwareBreakpointManager(nullptr),
       _hardwareBreakpointManager(nullptr), _terminated(false) {}
 
-Process::~Process() { terminate(); }
+Process::~Process() = default;
```

Comment:
Yeah but that is (or should be) in the destructor of the PTrace class.

On Thursday, April 28, 2016, strager <notifications@github.com> wrote:

> In Sources/Target/FreeBSD/Process.cpp
> <https://github.com/facebook/ds2/pull/304#discussion_r61446900>:
>
> > @@ -40,7 +40,7 @@ Process::Process()
> >      : super(), _softwareBreakpointManager(nullptr),
> >        _hardwareBreakpointManager(nullptr), _terminated(false) {}
> >
> > -Process::~Process() { terminate(); }
> > +Process::~Process() = default;
>
> Should we at least ptrace detach if we are attached?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/facebook/ds2/pull/304/files/841e4756d4e67a1f6cb7272361581735fd9d0a66..0492a25be66bb5c62995c0e63a01105be34f4938#r61446900>
>


-- 
Stephane Sezer